src/renderer/i18n/locales/it-IT.js

```diff
@@ -0,0 +1,964 @@
+export default {
+  COMMON: {
+    ADDRESS: 'Indirizzo',
+    ALL: 'Tutto',
+    APP_NAME: 'ARK Desktop Wallet',
+    APP_NAME_SHORT: 'ARK Desktop',
+    APPEARANCE: 'Aspetto',
+    AVATAR: 'Avatar',
+    BACK: 'Indietro',
+    BACKGROUND: 'Background',

```

Comment:
Is this correct?

Reply:
It depends.
If we mean background as "wallpaper" it should be:
BACKGROUND: "Sfondo".

If we mean for example that the application works in the background, then in this case it is correct:
BACKGROUND: "Background"

Reply:
It's for the "sfondo".